src/css/styles.css

```diff
@@ -0,0 +1,547 @@
+:root {
+  --very-light-purple: hsl(240, 100%, 96%);
+  --light-purple: hsl(235, 100%, 83%);
+  --medium-purple: hsl(234, 100%, 75%);
+  --dark-purple: hsl(234, 100%, 69%);
+  --color-border: hsl(218, 13%, 48%);
+  --color-icon: hsl(218, 20%, 39%);
+  --color-nav-p: hsl(218, 31%, 30%);
+  --color-heading-mobile: hsl(218, 54%, 20%);
+  --color-heading-desktop: hsl(218, 76%, 15%);
+  --light-blue: hsl(234, 97%, 62%);
+  --bright-blue: hsl(235, 72%, 48%);
+  --color-white: hsl(0, 0%, 100%);
+  --bg-body: hsl(0, 0%, 90%);
+  --color-light-gray: hsl(217, 13%, 74%);
+  --color-smoke-white: hsl(240, 100%, 96%);
+  --card-border: hsl(218, 14%, 89%);
+  --fs-tiny: 0.75em;
+  --fs-100: 0.8215em;
+  --fs-200: 0.9375em;
+  --fs-300: 1em;
+  --fs-400: 1.0625em;
+  --fs-500: 1.125;
+  --fs-600: 1.25em;
+  --fs-700: 2em;
+  --fs-800: 2.5em;
+  --fs-900: 3em;
+
+  font-family: "P", sans-serif;
+  font-size: 1rem;
+
+  --fw-lighter: 400;
+  --fw-regular: 500;
+  --fw-semi-bold: 600;
+  --fw-bold: 700;
+  --size-100: 0.5em;
+  --size-200: 1em;
+  --size-300: 1.5em;
+  --size-400: 2em;
+  --size-500: 2.5em;
+  --size-600: 3em;
+  --size-700: 4em;
+  --size-800: 6em;
+  --size-900: 7em;
+  --size-xl: 9em;
+}
+
+/* Box sizing rules */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+/* Remove default margin */
+* {
+  margin: 0;
+  padding: 0;
+  font: inherit;
+}
+
+/* Remove list styles on ul, ol elements with a list role, which suggests default styling will be removed */
+ul,
+ol {
+  list-style: none;
+}
+
+/* Set core root defaults */
+html {
+  scroll-behavior: smooth;
+  scroll-padding: 90px;
+}
+
+/* A elements that don't have a class get default styles */
+a:not([class]) {
+  text-decoration-skip-ink: auto;
+}
+
+/* Make images easier to work with */
+img,
+picture {
+  max-width: 100%;
+  display: block;
+}
+
+/* Remove all animations, transitions and smooth scroll for people that prefer not to see them */
+@media (prefers-reduced-motion: reduce) {
+  html:focus-within {
+    scroll-behavior: auto;
+  }
+
+  *,
+  *::before,
+  *::after {
+    animation-duration: 0.01ms !important;
+    animation-iteration-count: 1 !important;
+    transition-duration: 0.01ms !important;
+    scroll-behavior: auto !important;
+  }
```

Comment:
- [Optional] Please avoid frequently using the `!important` rule in your CSS as this is [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).